conda/README.md

```diff
@@ -0,0 +1,192 @@
+* Environments: If desired, every user can have a different set of packages, because conda can install & manage packages in personal folders instead of system ones.
+                A user can even have multiple such environments, and switch between them rapidly to work with different sets of related software without conflict.
+
+
+## Initial Conda Setup
+
+If you have not installed Conda yet, it can be downloaded as an all-in-one package that has no extra dependencies, known as "miniconda".
+For Linux, Unix, and Windows, this can be downloaded at https://docs.conda.io/en/latest/miniconda.html
+For z/OS, Conda can be downloaded from Rocket Software at https://www.rocketsoftware.com/zos-open-source
+
+Conda will prompt during the install for certain setup options, 
+And ultimately you'll want to put some conda initialization content into your startup script
+So that whenever you open your terminal, conda will be ready for your use.
+
+Once you have conda downloaded and installed, you'll want to create your first conda "environment"
+This can be done by providing a path or a nickname
+
+`conda create --prefix PATH`
+`conda create --name ENVIRONMENT`
+
+Either will work, but path helps you better seperate your content from content others use by placing it in a folder that you can have stricter permissions on.
+
+If you need to know more about certain commands, you can use the help command for any.
+
+`conda create --help`
+
+Or, check the official documentation: https://docs.conda.io/en/latest/index.html
+
+Once you have an environment, you should activate it so that the actions you do are on that environment, as opposed to the base one.
+
+`conda activate PATH_OR_NAME`
+
+Conda will detect whether the parameter is a path or a nickname, so this command works for both.
+
+Finally, you can view the conda environment and other information by checking "info"
+`conda info`
+
+## Managing Conda Channels
+
+When downloading a package, such as a Zowe Plugin, the place that you download from is configurable.
+These are called "Channels", but are very similar to "Repositories" seen in other package managers.
+With Conda, you can install from:
+
+* A network channel (Internet or company internal)
+* A local channel (Collection of plugins on your computer)
+* Just an individual package, without a channel
+
+You can have multiple of each, and if a package is present in more than one location, you can specify which one to use.
+
+## Searching for Packages
+
+Conda has a search utility that checks all the Channels, but it's important to note that because any type of software
+Can be installed through Conda, that you probably want to search through the details to help identify which ones are meant for Zowe,
+Or use Channels that are distinctly for Zowe so that you can get packages that are strictly for Zowe.
+
+`conda search anything_you_want`
+`conda search --info this_gives_more_details`
```

Comment:
this_gives_more_details should still be anything_you_want, since that's what the arg describes,

But you can probably change it to something like, 

`Conda has a search utility that searches for all Channels, `

`conda search anything_you_want`

`but it's important to note that because any type of software can be installed through Conda, you probably want to search through a detailed view to help identify which ones are meant for Zowe.`

`conda search --info anything_you_want`